api/lib/httpd_functions.pl

```diff
@@ -33,6 +33,7 @@ sub list_features
         'rh-php71-php-fpm' => { installed => (-f '/etc/e-smith/db/configuration/defaults/rh-php71-php-fpm/type') ? JSON::true : JSON::false, packages => ['nethserver-rh-php71-php-fpm'] },
         'rh-php72-php-fpm' => { installed => (-f '/etc/e-smith/db/configuration/defaults/rh-php72-php-fpm/type') ? JSON::true : JSON::false, packages => ['nethserver-rh-php72-php-fpm'] },
         'rh-php73-php-fpm' => { installed => (-f '/etc/e-smith/db/configuration/defaults/rh-php73-php-fpm/type') ? JSON::true : JSON::false, packages => ['nethserver-rh-php73-php-fpm'] },
+        'php80-php-fpm' => { installed => (-f '/etc/e-smith/db/configuration/defaults/php80-php-fpm/type') ? JSON::true : JSON::false, packages => ['nethserver-remi-php80-php-fpm'] },
```

Comment:
Even if not completely correct with the Remi package, I'd retain the `rh-` prefix: existing code looks more consistent with it and requires less changes. As alternative we could introduce a new repo prefix (say, `rm-`) specific to Remi.

```suggestion
        'rh-php80-php-fpm' => { installed => (-f '/etc/e-smith/db/configuration/defaults/rh-php80-php-fpm/type') ? JSON::true : JSON::false, packages => ['nethserver-remi-php80-php-fpm'] },
```

Reply:
We cannot do this, the service name is `php80-php-fpm`